### PR TITLE
Create Terraform modules

### PR DESCRIPTION
**Pull request by dtrendiuk**

### CODE REVIEW

File: Terraform/alb.tf [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # ALB
 resource "aws_alb" "dev_pro_alb" {
   name            = var.alb_name
-  subnets         = [aws_subnet.dev_pro_public_subnets[0].id, aws_subnet.dev_pro_public_subnets[1].id]
-  security_groups = [aws_security_group.dev_pro_sg_public.id]
+  subnets         = [module.vpc.public_subnet_ids[0], module.vpc.public_subnet_ids[1]]
```
Comment (DmytroMorhulDevPro) [inline Terraform/alb.tf]: `subnets = module.vpc.public_subnet_ids` - ?

File: Terraform/modules/ec2/ec2.tf [hunk 2]
```diff
@@ -1,8 +1,10 @@
+# EC2 instances
```
Comment (DmytroMorhulDevPro) [inline Terraform/modules/ec2/ec2.tf:1]: How to create 1 EC2 using your module?

File: Terraform/alb.tf (hunk 1 above)
Comment (dtrendiuk) [reply Terraform/alb.tf]: In this block I specify public_subnet_ids from outputs of vpc-module.

File: Terraform/modules/ec2/ec2.tf (hunk 2 above)
Comment (dtrendiuk) [reply Terraform/modules/ec2/ec2.tf:1]: In this case the only option to create a new EC2 instance is to add new resource to the main block.
Alternatively EC2 module can be changed by adding a new resource, or the whole logic of the EC2 module should be changed.

File: Terraform/alb.tf (hunk 1 above)
Comment (DmytroMorhulDevPro) [reply Terraform/alb.tf]: module.vpc.public_subnet_ids[0], module.vpc.public_subnet_ids[1] are in the same list which is module.vpc.public_subnet_ids why you added them separately?

**Comment by DmytroMorhulDevPro** [comment]
in general I'm ok to merge

**Pull request merged**